opendevin/server/listen.py

```diff
@@ -354,6 +355,21 @@ async def get_agents():
     return agents
 
 
+@app.get('/api/options/security-analyzers')
+async def get_security_analyzers():
+    """Get all security analyzers supported by the agent.
```

Comment:
```suggestion
    """Get all supported security analyzers.
```

Just a minor thing, it doesn't depend on the agent, right?

Reply:
you're right, it doesn't